moto/resourcegroupstaggingapi/models.py

```diff
@@ -455,6 +460,26 @@ def format_tag_keys(
 
                 yield {"ResourceARN": f"{kms_key.arn}", "Tags": tags}
 
+        # LexV2
+        lex_v2_resource_map: Dict[str, Dict[str, Any]] = {
+            "lexv2:bot": self.lexv2_backend.bots,
```

Comment:
Note that `self.lexv2_backend` may become None here, if it does not exist in this region, so we should guard against that here as well